components/SendCardAmount.vue

```diff
@@ -19,7 +19,12 @@ const balance = computed(() => {
   return '0'
 })
 
-const handleKeyDown = (customEvent: CustomEvent) => {
+/**
+ * Key down handled does input field validation
+ *
+ * @param customEvent
+ */
+const handleKeyDown = async (customEvent: CustomEvent) => {
```

Comment:
That's not good, why keydown. This will not work with paste then. I think the input event is better. Also the char down events don't match identically between mobile and desktop.

Reply:
Ok I need to make bigger refactor I guess, `input` event makes total sense.